services/Group.js

```diff
@@ -0,0 +1,75 @@
+const GroupEntity = require('../entities/Group');
+const SportEntity = require('../entities/Sport');
+const UserEntity = require('../entities/User');
+
+const Group = require('../models/groups');
+
+const createGroup = async(token, photo, name, sport, maxMembers, genders, levels, ageMin, ageMax, description, label, latitude, longitude, status) => {
+
+    const sportData = await SportEntity.findSportBySportName(sport);
+    const userData = await UserEntity.findUserByToken(token);
+
+    if(!sportData || !userData) {
+        return;
+    };
+
+    const newGroup = await new Group({
+        admin: userData._id,
+        photo,
+        name,
+        sport: sportData._id,
+        maxMembers,
+        genders,
+        levels,
+        ageMin,
+        ageMax,
+        description,
+        workout_location: {
+            label,
+            location: {
+                type: 'Point',
+                coordinates: [longitude, latitude]
+            }
+        }
+    });
+
+    const newGroupData = await newGroup.save();
+
+    await UserEntity.addUserToGroup(token, newGroupData._id, status);
+
+    return newGroupData;
+};
+
+const searchGroup = async(sport, latitude, longitude) => {
+
+    const sportData = await SportEntity.findSportBySportName(sport);
+
+    if(!sportData) {
+        return;
+    };
+
+    const groupsData = await GroupEntity.findGroupBySportAndOrLocation(sport, sportData._id, latitude, longitude);
+
+    return groupsData;
+};
+
+const getGroupInformation = async(group_id) => {
```

Comment:
Better to use camelCase in the code (same in `getGroupMembers`) !

Reply:
group_id

Reply:
Done!